src/components/NewBeer.js

```diff
@@ -0,0 +1,14 @@
+import { Routes, Route, NavLink } from 'react-router-dom';
+
+
+function NewBeer() {
+    return (
+        <>
+            <header>
+                <NavLink to="/">Home</NavLink>
+            </header>
```

Comment:
That's exactly why you created `Header` component! You should have this link there, instead fo writing this piece of code in every component.:)